src/components/quote/code/templates/quoteTemplate.jsx

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var classNames = require('classnames');
+var getComponentClasses = require('../../../../utils/getComponentClasses');
+
+module.exports = function() {
+
+  var propClasses = ['size', 'purpose', 'type'];
+  var classes = getComponentClasses('component-quote', propClasses, this.props);
+
+  return (
+      <q className={classNames(classes)} cite={this.props.cite} itemScope itemType="http://schema.org/CreativeWork" itemProp="text">
+        {this.props.children}
+        {(this.props.cite) ? <meta itemProp="citation" content="{this.props.cite}" /> : null}
```

Comment:
should the double quotes be removed here: `content="{this.props.cite}"` 
The output in the html is this as a string: http://grab.by/IDdi